core/new-gui/src/app/workspace/component/result-panel/result-panel.component.ts

```diff
@@ -38,7 +38,7 @@ export class ResultPanelComponent implements OnInit {
   frameComponentConfigs: Map<string, ResultFrameComponentConfig> = new Map();
 
   // the highlighted operator ID for display result table / visualization / breakpoint
-  currentOperatorId?: string;
+  currentOperatorId?: string | undefined;
```

Comment:
is this needed? Isn't `x?: T` equipment to `x: T | undefined`?

Reply:
either way is fine, `| undefined` is just more explicit.

Reply:
ok